paper.md

```diff
@@ -58,202 +58,195 @@ bibliography: paper.bib
+technique used successfully in workshops developed by The Carpentries
+[@wilson-software-carpentry]. This technique involved having at least two
+instructors present for each lesson, where one instructed and another acted as a
+"helper". Students would signal for assistance by attaching colored sticky notes
+to the back of their laptop monitor. This method avoided interrupting the lesson
+flow when individual students needed assistance.
 
 ## Story of the project
 
-While there are many excellent open source libraries for quantitative data
-analysis, the use of less capable analysis tools (such as spreadsheet software)
-is still prevalent among researchers although these drastically reduce
-analysis reproducibility, power, and efficiency. Partly this happens because of
-lack of awareness, and partly because students are often required to learn
-about new tools in isolation and on top of their main research activities.
-Those that do embark on this daunting journey often quit before they can
-reap the rewards of their efforts. Like-minded peers could provide the needed
-support structure to uphold motivation, but are often few and far between,
-especially in fields without a strong quantitative culture. To remedy these
-issues, we launched the student group [UofT
-Coders](http://uoftcoders.github.io/) which teaches how to use code for
-research through skill sharing, co-working and community building in a friendly
-environment.
-
-After receiving overwhelmingly positive feedback on our content and teaching style,
-we sought to formally share our experiences through the university
-curriculum. We designed a course on open, reproducible data analysis, which we
-taught as a fourth-year undergraduate course in the Department of
-Ecology and Evolutionary Biology with the title "Theoretical Ecology and
-Reproducible Quantitative Methods in R." We modelled the structure and portions
-of the course content after ["Reproducible Quantitative
-Methods"](https://cbahlai.github.io/rqm-template/) a course created by Dr.
-Christie Bahlai, modifying the lesson content to include additional theoretical
-ecology topics but maintaining the focus on reproducibility and computational
-skills.
-
-Following a successful pilot term we modified our lecture material to include more generally applicable
-statistical concepts and fewer theoretical ecology concepts, and renamed the
-course "Quantitative Methods in R for
-Biology" [@rcourse] to reflect this change. On
-both occasions, the course received excellent feedback from the students and
-the supervising professors.
+While there are many excellent open source software available for quantitative
+data analysis, the use of less capable tools (such as spreadsheet software)
+is still prevalent among researchers, even though these drastically reduce
+analytical reproducibility, power, and efficiency. This happens partly due to
+lack of awareness, and partly because students (who will be future researchers)
```

Comment:
Should we assume that all students in the course will be future researchers? Maybe change to "many of whom will be future researchers".

Reply:
I think this is referring more generally to grad students, not students in the course. It might be a good idea to change 'students' to 'graduate students' for clarity and remove the bit in parentheses entirely.